src/Sulu/Bundle/LocationBundle/Resources/config/geolocator.xml

```diff
@@ -9,19 +9,21 @@
 
     <services>
         <!-- Guzzle client -->
-        <service id="sulu_location.geolocator.guzzle.client" class="%sulu_location.guzzle.client.class%"/>
+        <service id=""sulu_location.geolocator.guzzle.client" class="%sulu_location.guzzle.client.class%">
```

Comment:
```suggestion
<service id="sulu_location.geolocator.guzzle.client" class="%sulu_location.guzzle.client.class%">
```

There is a acccidently `""`

Reply:
yes thout to edit that very quickly. Always a bad idear to push unchecked code. fixed this now.